packages/common/sass/components/_popover.scss

```diff
@@ -1,6 +1,6 @@
 @import './_variables';
 
-.popover {
```

Comment:
This class name was causing some conflict with PopperJS.

Reply:
What are the conflicts? Renaming this would be a breaking change.